### ISSUE TITLE
Componente `Carousel`: verificare cose di accessibilità

### ISSUE THREAD

**Issue by Fupete** [italia/bootstrap-italia#1161]
## Cosa

Durante le verifiche di accessibilità sono emerse alcune cose a priorità low e medium da verificare: 

- [x] 1. [medium] le frecce laterali, una volta premute, secondo me dovrebbero spostare il focus sul primo elemento della slide/pagina di atterraggio, così come è il comportamento dei pallini di indice, permetterebbe una navigazione più agevole da lettore di schermo;
- [x] 2. [medium] il "bordo" del focus viene tagliato, in basso e talvolta anche in alto o di lato;
- [x] 3. [low] le aria-label delle freccie laterali indicano "**slide**", mentre i cerchietti parlano di "**pagina**", bisogna uniformare anche per corrispondere al disclaimer accessibilità presente nella documentazione di BSI;
- [ ] 4. [low] valutare di ingrandire i "cerchietti" e il loro spazio a disposizione per renderli più usabili, da fare in caso partendo da UI Kit Italia;
- [x] 5. [low] dovremmo probabilmente cercare di non saltare livelli di heading nei titoli presenti negli esempi (seppur consci che le card cambieranno). 

## Perché

Amichevole con i lettori di schermo e Navigabile. Documentazione chiara. 

## Contesto

Verifiche di accessibilità sui componenti.

⚠️ Sarà da aggiornare lo status delle verifiche nella [board census ](https://github.com/orgs/italia/projects/32/views/1)alla risoluzione.

## Altro

Da verificare l'impatto di possibile e auspicabile upgrade all'ultima versione di Splide 4, come già tracciato in questa issue #1122. 

Da verificare, uniformare, anche in Angular kit. 
In React Kit al momento non è presente, seppur c'è qui una issue (chiusa da Stale) per la sua creazione https://github.com/italia/design-react-kit/issues/395 (cc @astagi)

Chiederei un parere sulle indicazioni che precedono anche a @cfabry con cui ne facemmo una prima revisione tempo fa.

**Comment by cfabry** [comment]
Concordo su tutto @Fupete, ma aggiungo alcune cose:
1. Con in codice attuale non abbiamo alcuna semantica che racchiuda l'oggetto e raggruppi insieme le varie parti (abbiamo solo un div contenitore): potremmo ipotizzare di inserire un elemento `<section>` come contenitore e descriverlo con _aria-roledescription_; c'è poi da capire quale valore dare ad _aria-roledescription_, perché "carousel" è molto chiaro per gli esperti ma per molte persone potrebbe non significare nulla. NOTA: il fatto che esista un titolo del carousel non è influente rispetto a roledescription perché il titolo non comunica il funzionamento della struttura.
2. Dentro al carousel abbiamo in sostanza le card e anche qui abbiamo un po' il problema dell'ordine di lettura quando il titolo non è il primo elemento. 
3. In questo tipo di carousel i "cerchietti" sono una lista `<ul>` semplice, e l'informazione semantica del loro funzionamento è fornita soltanto dall'aria-label del button. Nel Carousel del [progetto APG](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/) del W3C la struttura è creata con elementi di ruolo tab, tablist, tabpanel pertanto molto più esplicito. Per sopperire a questa problematica è forse necessario indicare più marcatamente il funzionamento di quei pallini; si potrebbe valutare di inserire in un elemento <nav>, visto che rappresentano la navigazione delle slide, con relativo aria-label. Tutto questo per "simulare" quello che fa in APG il ruolo "tablist" con la sua aria-label.
4. Dovremmo valutare la possibilità di inserire una auto-rotation, non tanto per suggerire tale tecnica che, viceversa, dobbiamo scoraggiare, quanto per rimarcare che, qualora esista, è obbligatorio includere un pulsante esplicito di stop dello scorrimento automatico.

**Comment by Fupete** [comment]
Grazie @cfabry per le considerazioni, molto utili. 

Reminder (anche da segnalazione pervenuta su Slack canale #design-system): aggiungerei alla presente issue da verificare le discrepanze e l'attinenza con gli esempi di [aria-pattern standard W3C](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/) del Carousel, potrebbe essere utile specialmente per l'uso di `role` specifici sugli item e sui contenitori.

**Comment by Fupete** [comment]
Rianalizzando la issue e le diverse discussioni direi che intanto l'aggiornamento a Splide v4 del plugin che usiamo (e relativo setting) è il primo passo da fare, avendo questo "risolto" molte cose in se. Qui alcuni improvement e cose aperte https://splidejs.com/guides/accessibility/ cc @astagi @zetareticoli . 

Mettiamolo in priorità per inizio anno.

**Comment by astagi** [comment]
@Fupete @cfabry l'aggiornamento di Splide si può trovare nella PR #1401

**Comment by Fupete** [comment]
Ho fatto un check qui: https://github.com/italia/bootstrap-italia/pull/1401#pullrequestreview-2623847205

A occhio e croce la versione nuova è molto meglio, rimangono un po' di cose
